docs/src/pages/docs/events.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Events
+description: Anchor - Events
+---
+
+Events are an incredibly powerful feature in Anchor. Rather than polling or refreshing to
+find out if an account's state has changed, events are like callbacks that save space, compute,
+and enable asynchronous programming.
+
+The main downside to events is that they are base64 encoded, and therefore not human readable.
+Despite this tradeoff, the UI can easily decode them, and the user benefits from this 
+compact mode of logging. There are many use cases for events, ranging from simple return values, to compact storage storage solutions for data logging.
+
+## Event Use Cases
+
+### Return Values
+We can use an event to immediately provide a return value, such as details related
+to a transaction. Since calls to certain programs only return a tx hash, this may not provide sufficient detail to your UI. An event here can provide a richer experience, by returning values about how a particular transaction was executed, for example.
+
+### Callbacks
+ You may want to monitor an event and be notified only once a transaction is confirmed.
+Instead of polling / querying the blockchain and looking for an account's values to update, we can build our client to only act when the program has emitted an event, saving on compute resources and fees.
+
+### Compact Storage 
+Since events are stored in base64, they can be an economical way to store data
+without the trouble of account creation and provisioning for rent-exemption, for example.
+
+## Example
+
+The simplest way to use events is to tack them onto the tail of your program's functions, allowing them to emit
+structured data when the function completes without error. 
+
+Start by creating a new anchor project:
+```bash
+$ anchor init Events
```

Comment:
```suggestion
anchor init Events
```

In other sections of the docs we don't add the $. I agree with upholding that standard as I've always found the `$` to get in the way when I'm trying to copy it.